src/Utils/functions.jsx

```diff
@@ -62,29 +62,31 @@ export function showTextToRaw(text) {
   return rawWords.join("_");
 }
 export function breakLongLines(sentence, chunkSize) {
```

Comment:
Maybe add a cooment about the purpose of the function, because it has a mix of strings and html, so it's a bit confusing

Reply:
ok